yaptide/celery/tasks.py

```diff
@@ -1,15 +1,20 @@
 import contextlib
 import logging
-import os
+import multiprocessing
 import tempfile
 from datetime import datetime
 from pathlib import Path
 
-import eventlet
-
 from yaptide.admin.simulators import SimulatorType, install_simulator
-from yaptide.celery.utils.pymc import (average_estimators, read_file, run_fluka,
-                                       run_shieldhit)
+sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssfrom yaptide.celery.utils.pymc import (average_estimators,
```

Comment:
Something strange

Reply:
Yes, will fix soon, I've also merged and included here Fluka runner, which made this code into not-working  state

Reply:
Also - this branch was rebased couple of times, remove your local branch before dealing with it